modules/db/src/main/scala/gem/dao/StepDao.scala

```diff
@@ -141,22 +143,24 @@ object StepDao {
     sql"""
       SELECT s.instrument,
              s.step_type,
-             sg.continuum,
-             sg.ar_arc,
-             sg.cuar_arc,
-             sg.thar_arc,
-             sg.xe_arc,
-             sg.filter,
-             sg.diffuser,
-             sg.shutter,
-             sg.exposure_time,
-             sg.coadds,
+             gc.continuum,
+             gc.ar_arc,
+             gc.cuar_arc,
+             gc.thar_arc,
+             gc.xe_arc,
+             gc.filter,
+             gc.diffuser,
+             gc.shutter,
+             gc.exposure_time,
+             gc.coadds,
              sc.offset_p,
              sc.offset_q,
              ss.type
         FROM step s
              LEFT OUTER JOIN step_gcal sg
```

Comment:
The join will be much more efficient than doing a subquery for each step. I was thinking we would do a second query for the instrument slices because it will be the same instrument for the whole sequence. So, 2 selects rather than N.